test/public/logs/overview.test.js

```diff
@@ -138,20 +142,19 @@ module.exports = () => {
 
         // Open the filters
         await pressElement(page, '#openFilterToggle');
-        await waitForTimeout(20);
 
         // Insert some text into the filter
-        await page.type('#contentFilterText', 'particle');
-        await waitForTimeout(300);
+        await fillInput(page, '#contentFilterText', 'particle');
+        await page.waitForFunction(() => document.querySelectorAll('table tbody tr').length === 2, { timeout: 1500 });
```

Comment:
Very good! I would advise to extract a function for this, for example "waitForTableToLength(size)", for example:

```js
module.exports.waitForTableToLength = async (page, expectedSize) => page.waitForFunction(
    (expectedSize) => document.querySelectorAll('tbody tr:not(.loading-row):not(.empty-row)').length === expectedSize,
    { timeout: 500 },
    expectedSize,
);
```

(once https://github.com/AliceO2Group/Bookkeeping/pull/1559#pullrequestreview-2076322710 has been merged, because it adds the selector for the no-data table)

Reply:
Done, added timeout as param with default 500ms